internal/pkg/service/cli/cmd/local/template/use.go

```diff
@@ -58,13 +58,17 @@ func UseCommand(p dependencies.Provider) *cobra.Command {
 
 			// Use template
 			opResult, err := useOp.Run(d.CommandCtx(), projectState, template, options, d)
+			if err != nil {
+				return err
+			}
 
 			if len(opResult.Warnings) > 0 {
 				for _, w := range opResult.Warnings {
 					d.Logger().Warn(w)
 				}
 			}
-			return err
+
+			return nil
```

Comment:
Tu bola chyba, `opResults` su `nil` ak nastane chyba, preto nemozeme z nich vypisovat `Warnings`.
Ak nastane chyba ma sa hned vratit, warning sa nevypisuju.

Reply:
Stane sa to, ak je template nevalidna - nepodarilo sa ju nacitat, vid novy E2E test.